packages/shared/src/utils/__test-sample.ts

```
@@ -0,0 +1,37 @@
+
+export function getItems<T>(data: T[]): T[] {
+  return [...data];
+}
+
+export function formatName(first: string, last: string) {
+  return first + " " + last;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function processData(data: any) {
+  var results = [];
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)